runtime/testy/Ansible.py

```diff
@@ -142,7 +142,7 @@ def udpSender(self, badThingsQueue, stateQueue, pipe):
                         s.sendto(msg, (host, UDPSendClass.SEND_PORT))
                     nextCall += 1.0/self.socketHZ
                     time.sleep(max(nextCall - time.time(), 0))
-                except Exception:
+                except Exception as e:
```

Comment:
Whats the proper procedure here? Should i mention it in the commit/ pull request message?